CDCgov/prime-reportstream
# New receivers for Nevada

This PR creates new receivers for Nevada. Nevada is changing vendors. The new vendor has requested test and production receivers.

Test Steps:
1. set new settings: `./prime multiple-settings set --input ./settings/organizations.yml`
2. create fake simple report file for NV: `./prime data --input-fake 2 --input-schema primedatainput/pdi-covid-19 --output <PATH TO FILE>.csv --target-states NV`
3. Open file and set processing mode code to `T` for one result. The other result should have a value of `P`
4. Run files through and ensure one result goes to prod receiver and other result goes to test receiver

## Changes
- added elr-test reciever
- added scripts to /staging and /prod folders

## Checklist

### Testing

## Linked Issues
- Fixes #6463
